src/server/pps/cmds/defaults_test.go

```diff
@@ -249,3 +249,61 @@ func TestCreatePipeline_delete(t *testing.T) {
 		"RepoName", "input",
 		"PipelineName", "pipeline").Run())
 }
+
+func TestCreatePipeline_leading_zero(t *testing.T) {
```

Comment:
what do you think of just having a unit test for yamlToJson instead of this integration test?

Reply:
I implemented this test as a regression test for the user-experienced bug. But you’re right that a unit test for the function itself would be good … I’ll add one.